gateway/uplink.go

```diff
@@ -33,23 +33,32 @@ func (g *gateway) parseDataUplink(ctx context.Context, phyPayload []byte, packet
 
 	g.logger.Debugf("received uplink from %s", device.NodeName)
 
-	// we will send one device downlink from the do command per uplink.
+	sendAck := false
+	// confirmed data up, send ACK bit in downlink
+	if phyPayload[0] == 0x80 {
```

Comment:
as a part of the refactor we should try replacing all of these (`0x80`) with constants. Would hopefully make the code easier to read and more obvious what we are checking for in different spots

Reply:
Whatever linters you guys turn on will have a field day.

Reply:
eh rdk turns off magic number linter too so I don't see us using that one